config/jobs.txt

```diff
@@ -41,9 +41,9 @@ Roboticist=2,2
 
 Warden=1,1
 Detective=1,1
-Security Officer=10,8
```

Comment:
Not sure why this was at 10, someone must have got a little overzelous with security and not noticed the security scaling coefficient.